commands/volumes/volume-create.go

```diff
@@ -3,26 +3,24 @@ package volumecommands
 import (
 	"errors"
 	"net/http"
-	"os"
 
 	gderrors "github.com/gluster/glusterd2/errors"
-	"github.com/gluster/glusterd2/gdctx"
 	restutils "github.com/gluster/glusterd2/servers/rest/utils"
 	"github.com/gluster/glusterd2/transaction"
 	"github.com/gluster/glusterd2/utils"
-	"github.com/gluster/glusterd2/volgen"
 	"github.com/gluster/glusterd2/volume"
 
 	"github.com/pborman/uuid"
 )
 
 // VolCreateRequest defines the parameters for creating a volume in the volume-create command
 type VolCreateRequest struct {
-	Name         string   `json:"name"`
-	Transport    string   `json:"transport,omitempty"`
-	ReplicaCount int      `json:"replica,omitempty"`
-	Bricks       []string `json:"bricks"`
-	Force        bool     `json:"force,omitempty"`
+	Name         string            `json:"name"`
+	Transport    string            `json:"transport,omitempty"`
+	ReplicaCount int               `json:"replica,omitempty"`
+	Bricks       []string          `json:"bricks"`
+	Force        bool              `json:"force,omitempty"`
+	Options      map[string]string `json:"options,omitempty"`
```

Comment:
Cool, it is added advantage to accept volume create and options together.

commented just for clarification.